### PR TITLE
None/null functionality and test cases 

### PR DESCRIPTION
**Pull request by cjoshea9**

### CODE REVIEW

File: gast_to_code.py [hunk 1]
```diff
@@ -84,6 +84,8 @@ def gast_router(gast, out_lang):
         return '"' + gast["value"] + '"'
     elif gast["type"] == "bool":
         return out["bool"][out_lang](gast)
+    elif gast["type"] == "none":
+        return "None" if out_lang == "py" else "null"
```
Comment (g-harel) [inline gast_to_code.py:88]: Should probably move this out of here to keep up the pattern.

Comment (g-harel) [inline gast_to_code.py:88]: Are we making a distinction between null and undefined in JS?

Comment (Danish21) [reply gast_to_code.py:88]: +1. If not, I would add a comment stating that we are arbitrarily choosing null.

Comment (cjoshea9) [reply gast_to_code.py:88]: We have decided to not worry about undefined for now and could possibly add in later. I will add a comment.

**Pull request merged**